cameo/strain_design/heuristic/evolutionary/optimization.py

```diff
@@ -806,3 +843,205 @@ def __init__(self, reaction_db=None, *args, **kwargs):
 
     def run(self, **kwargs):
         KnockinKnockoutOptimization.run(self, **kwargs)
+
+
+class CofactorSwapOptimization(KnockoutOptimization):
```

Comment:
column name results fixed, and plot raises `NotImplementedError`. I can't reproduce the problem with the non-appearing plot, however that and the missing `plot` function is older code that needs fixing so can work on that in a bug fix issue (e.g. not even sure we need that plot function so would rather drop it but that depends on what the original intent was).